autogpt_platform/backend/backend/server/v2/store/db.py

```diff
@@ -809,7 +809,8 @@ async def get_agent(
     try:
         store_listing_version = (
             await prisma.models.StoreListingVersion.prisma().find_unique(
-                where={"id": store_listing_version_id}
+                where={"id": store_listing_version_id},
+                include={"Agent": True},
```

Comment:
By not including it, we’re intentionally keeping the store listings implementation under wraps. All the necessary details are already stored in the store tables.